core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -378,6 +378,19 @@ private static void sendConfirmation(
         readyForNextRequest(context);
     }
 
+    private static void sendInsertConfirmation(
+            JsonQueryProcessorState state,
+            CharSequence keepAliveHeader
+    ) throws PeerDisconnectedException, PeerIsSlowToReadException {
+        final HttpConnectionContext context = state.getHttpConnectionContext();
+        final HttpChunkedResponseSocket socket = context.getChunkedResponseSocket();
+        header(socket, keepAliveHeader, 200);
+        socket.put('{')
```

Comment:
it would be good to indicate INSERT was executed? maybe including the number of inserted rows if that's possible? It seems the `OperationFuture` provides that: `getAffectedRowsCount()` (I've never tried it so you would need to check it has the expected behaviour)

Reply:
The problem I'm having is that OperationFuture can be null and I'm not sure if it's null where am I supposed to get the count of the records that were inserted during the query 🤔